jsparagus/actions.py

```diff
@@ -211,23 +211,29 @@ class FunCall(Action):
     pushpathne non-terminal. The replay attribute of a reduce action correspond
     to the number of stack elements which would have to be popped and pushed
     again using the parser table after reducing this operation. """
-    __slots__ = 'trait', 'method', 'offset', 'args', 'set_to'
-    def __init__(self, trait, method, alias_read, alias_write, args, set_to = None, offset = 0):
+    __slots__ = 'trait', 'method', 'offset', 'args', 'fallible', 'set_to'
+    def __init__(self, trait, method, fallible, alias_read, alias_write, args, set_to = None, offset = 0):
```

Comment:
it's better making them keyword arguments.
it's not clear which one means what, in the callsite.